config/webpack.config.js

```diff
@@ -80,7 +80,7 @@ module.exports = env => {
       rules: [
         {
           test: /\.js$/,
-          exclude: /node_modules/,
+          exclude: /node_modules(?!\/@novnc)/,
```

Comment:
As someone still mostly unfamiliar with webpack: why is this needed?

Reply:
I believe its because of https://github.com/theforeman/foreman/pull/6056#issuecomment-423301264

Reply:
Those are the things that deserve an inline comment.

Reply:
Sure! Will do so tonight, should've done this right away. The reason is indeed as ohadlevy mentioned.